data_analysis/plots.py

```diff
@@ -72,111 +288,464 @@ def boxplot(
+        save_path = create_save_path(save_name)
+        logger.info(f'Saving to {save_path}')
+        fig.write_html(f'{save_path}.html')
+        fig.write_image(f'{save_path}.png')
+        logger.info('Saving to Chart Studio')
+        py.plot(fig, filename=f"{save_name}")
 
 
 def plot_map(
     geo_df: gpd.GeoDataFrame,
-    var: str,
     save_name: str,
-    cmap: str = None,
-    background_map: folium.Map = None,
-        save: bool = True) -> folium.Map:
+    save: bool = True,
+        **kwargs: dict) -> folium.Map:
     """ Create a map of bus routes from GeoDataFrame
 
     Args:
         geo_df (gpd.GeoDataFrame): DataFrame with bus routes, GPS coordinates,
            and some variable of interest e.g. ratio
-        var (str): the variable of interest for a color scale e.g. ratio
         save_name (str): The name of the saved output map.
-        cmap (str, optional): The color map to use for coloring the
-           variable of interest. Defaults to None.
-        background_map (folium.Map, optional): The background map for the
-           bus routes in the DataFrame. Defaults to None.
         save (bool, optional): Whether to save the map. Defaults to True.
-
+        **kwargs (dict): A dictionary of keyword arguments passed
+            to GeoDataFrame.explore
     Returns:
         folium.Map: A map of bus routes colored by a target variable.
     """
     geo_df = geo_df.copy()
-    newmap = geo_df.explore(
-        column=var,
-        cmap=cmap,
-        m=background_map,
-        legend=True)
+    # Convert dates to strings. Folium cannot handle datetime
+    date_columns = geo_df.select_dtypes(include=["datetime64"]).columns
+    geo_df[date_columns] = geo_df[date_columns].astype(str)
+    newmap = geo_df.explore(**kwargs)
     if save:
-        save_path = str(PLOTS_PATH / f'{save_name}_{var}.html')
-        logger.info(f'Saving {save_path}')
-        newmap.save(save_path)
+        save_path = create_save_path(f'{save_name}_{kwargs["column"]}')
+        logger.info(f'Saving to {save_path}')
+        newmap.save(f"{save_path}.html")
     return newmap
 
 
-def main() -> None:
-    """Generate boxplot, maps of all routes, top 10 best routes, and
-    top 10 worst routes
+def create_ridership_map(mvp: bool = False) -> None:
+    """Generate a map of ridership per route
+
+    Args:
+        mvp (bool, optional): Whether to generate the data and map
+            used in the MVP. Defaults to False.
     """
-    summary_df = compare_scheduled_and_rt.main()
+    # Daily totals by route
+    ridership_by_rte_date = pd.read_csv(
+        "https://data.cityofchicago.org/api/views/"
+        "jyb9-n7fm/rows.csv?accessType=DOWNLOAD")
 
-    gdf = static_gtfs_analysis.main()
+    ridership_by_rte_date['date'] = pd.to_datetime(
+        ridership_by_rte_date['date'],
+        infer_datetime_format=True
+    )
 
-    summary_gdf = summary_df.merge(gdf, how="right", on="route_id")
+    if mvp:
```

Comment:
`mvp` refers to the date range for the MVP i.e. `2022-05-20` to `2022-11-05`. I saved `csv`s of `combined_long_df` and `summary_df` from that run to aid reproducibility. 

These probably should be configurable, but I wanted to set aside those dates in particular so that the data used in the MVP is easily accessible.